### DISCUSSION TITLE
Exchanger geometry STL available ?

### DISCUSSION THREAD

**Discussion by yvanblanchard** [leap71/PicoGK#35] [General]
Hello,

I was wondering if this geometry is available for sharing (STL mesh) ?
I would like to test mesh-free techniques for FEA on it
(evn if I know that the advised approach consists at using the same input fields that were used for the design itself)

![image](https://github.com/leap71/PicoGK/assets/63931618/8991bbfa-b4a7-46e3-a6dc-92e7f0d5b86a)

**Comment by LinKayser** [comment]
Sorry, this one is unavailable, as it is a functioning aerospike rocket engine. We will open source more geometries in the future, but not that one.

Cheers,
Lin
